libcxx/test/std/ranges/range.adaptors/range.take/begin.pass.cpp

```diff
@@ -27,55 +28,104 @@ struct NonCommonSimpleView : std::ranges::view_base {
 static_assert(std::ranges::sized_range<NonCommonSimpleView>);
 static_assert(!std::ranges::sized_range<const NonCommonSimpleView>);
 
+using CommonInputIterPtrConstInt        = common_input_iterator<const int*>;
+using CountedCommonInputIterPtrConstInt = std::counted_iterator<CommonInputIterPtrConstInt>;
```

Comment:
FWIW, I'd use a shorter name (which IMO is fine since it's only used within a single file). It's a super minor comment that's not worth another around of review, so no action necessary. :)